app/styles/components/simple-chart-tooltip.scss

```diff
@@ -0,0 +1,10 @@
+.simple-chart-tooltip {
+  left: 13rem !important;
```

Comment:
Why is `!important` necessary here? It's my impression for posts like [this one](https://css-tricks.com/when-using-important-is-the-right-choice/) this this isn't a good idea.

Reply:
I tried several things, including being more explicit, and I couldn't get it to override for now.

Reply:
Ok, can you create a ticket as a reminder for us to get into this at some point. It's not critical, but it's probably worth doing.

Reply:
Done.